ethereum/blocks.py

```diff
@@ -1169,10 +1169,13 @@ def finalize(self):
         self.ether_delta += delta
 
         for uncle in self.uncles:
-            r = self.config['BLOCK_REWARD'] * \
-                (self.config['UNCLE_DEPTH_PENALTY_FACTOR'] + uncle.number - self.number) \
-                / self.config['UNCLE_DEPTH_PENALTY_FACTOR']
-            r = int(r)
+            br = self.config['BLOCK_REWARD']
+            udpf = self.config['UNCLE_DEPTH_PENALTY_FACTOR']
+            un = uncle.number
+            bn = self.number
+
```

Comment:
Those assignments (except `uncle.number`) could probably be pulled out of the loop (assuming `self.delta_balance()` doesn't modify self.number).